net-irc/irssi/irssi-1.1.1.recipe

```diff
@@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
 made available for Irssi to customise how it looks and operates. Plugins \
 are available which add encryption and protocols such as ICQ and XMPP."
 HOMEPAGE="https://irssi.org/"
-COPYRIGHT="1999-2016 Timo Sirainen"
+COPYRIGHT="1999-2018 Timo Sirainen"
 LICENSE="GNU GPL v2"
```

Comment:
An exception was also [granted][0].

> Specific permission is granted for the GPLed code in this distribution to
be linked to OpenSSL without invoking GPL clause 2(b).


[0]: https://github.com/irssi/irssi/blob/1.1.1/COPYING#L1

Reply:
The exception means that it's a customized version of GNU GPL v2, so please add a new license called "GNU GPL v2 with OpenSSL exception" and adjust `LICENSE` accordingly